core/math/FastRNG.h

```diff
@@ -26,6 +26,8 @@
 #define __FAST_RNG_H__
 
 #include "math/MathBase.h"
+#include <iostream>
```

Comment:
well, iostream not required

Reply:
for std::is_same

Reply:
I guess iostream replaced with type_traits?